src/mito2/src/sst/parquet/reader.rs

```diff
@@ -357,22 +353,27 @@ impl ParquetReaderBuilder {
         }
 
         let inverted_filtered = self
-            .prune_row_groups_by_inverted_index(row_group_size, parquet_meta, &mut output, metrics)
+            .prune_row_groups_by_inverted_index(row_group_size, &mut output, metrics)
```

Comment:
With the RowGroupSelection, maybe we can filter by min-max first now. But we may not change the behavior in this PR.